test/integration_test/main_test.go

```diff
@@ -51,6 +51,26 @@ func setup() error {
 		"upgrade-hops-url-list",
 		"",
 		"List of Portworx Spec Generator URLs separated by commas used for upgrade hops")
+	flag.StringVar(&ci_utils.CloudProvider,
+		"cloud-provider",
+		"",
+		"Type of cloud provider")
+	flag.StringVar(&ci_utils.PxEnvVars,
```

Comment:
I am assuming this will be used to specify the creds for cloud or vsphere, right?
If yes, I think it is okay to pass them as plain text for now, but soon we should change this and use a k8s secrets to pass them into the storagecluster.